.travis.yml

```diff
@@ -10,16 +10,16 @@ env:
     - FRAMEWORK_NAME="ReSwiftRouter"
     - UPDATE_DOCS="false"
 
-osx_image: xcode8.2
+osx_image: xcode10
```

Comment:
@DivineDominion this should be all that is needed to get it to work with travis in your PR. I updated the project settings and Swift version if wanted.